spec/controllers/organisers_controller_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+
+RSpec.describe OrganisersController, type: :controller do
+	describe 'Organisers Controller GET Tests' do
+		before(:each) do
+			@organiser = create(:user)
+		end
+
+		it 'should get the organiser dashboard' do
+			edition = create(:edition)
+			login_user(@organiser)
+			get :dashboard, user: @organiser
```

Comment:
we should check the content of the dashboard page too. otherwise we're not sure whether the page rendered is correct one

Reply:
Now we are checking the rendered file with